### PR TITLE
[WRR] Prefer application_utilization to cpu_utilization

### PR DESCRIPTION
**Pull request by yousukseung**

### CODE REVIEW

**Comment by yousukseung** [comment]
ORCA code is commented out yet.

**Review by markdroth — APPROVED** [review]
This looks good, modulo a few minor test comments.  Feel free to merge after addressing.

**Comment by markdroth** [comment]
Note that we can't merge this until the orca proto change has been imported.

File: test/core/client_channel/lb_policy/weighted_round_robin_test.cc [hunk 1]
```diff
@@ -334,24 +340,58 @@ TEST_F(WeightedRoundRobinTest, Basic) {
...
+  ASSERT_NE(picker, nullptr);
+  // Address 0 gets weight 1, address 1 gets weight 3.
+  // No utilization report from backend 2, so it gets the average weight 2.
+  WaitForWeightedRoundRobinPicks(
+      &picker,
+      {{kAddresses[0], MakeBackendMetricData(/*app_utilization=*/0,
+                                             /*qps=*/100.0, /*eps=*/0.0,
+                                             /*cpu_utilization=*/0.9)},
+       {kAddresses[1],
+        MakeBackendMetricData(/*app_utilization=*/0,
```
[the remark this answers is not in the document]
Comment (yousukseung) [reply test/core/client_channel/lb_policy/weighted_round_robin_test.cc:375]: Added a new test.

File: test/core/client_channel/lb_policy/weighted_round_robin_test.cc [hunk 2]
```diff
@@ -452,6 +492,50 @@ TEST_F(WeightedRoundRobinTest, OobReporting) {
...
+  auto picker = SendInitialUpdateAndWaitForConnected(
+      kAddresses, ConfigBuilder().SetEnableOobLoadReport(true));
+  ASSERT_NE(picker, nullptr);
+  // Address 0 gets weight 1, address 1 gets weight 3.
+  // No utilization report from backend 2, so it gets the average weight 2.
+  ReportOobBackendMetrics(
+      {{kAddresses[0], MakeBackendMetricData(/*app_utilization=*/0,
+                                             /*qps=*/100.0, /*eps=*/0.0,
+                                             /*cpu_utilization=*/0.9)},
+       {kAddresses[1], MakeBackendMetricData(/*app_utilization=*/0,
```
[the remark this answers is not in the document]
Comment (yousukseung) [reply test/core/client_channel/lb_policy/weighted_round_robin_test.cc:542]: Added a new test.

File: test/cpp/end2end/xds/xds_wrr_end2end_test.cc [hunk 3]
```diff
@@ -40,17 +40,61 @@ using ::envoy::extensions::load_balancing_policies::
...
+        return false;
+      }
+      num_picks = 0;
+      ResetBackendCounters();
+    }
+    return true;
+  });
+}
+
+TEST_P(WrrTest, CpuUtilWithNoAppUtil) {
```
[the remark this answers is not in the document]
Comment (yousukseung) [reply test/cpp/end2end/xds/xds_wrr_end2end_test.cc]: Removed.

**Comment by yousukseung** [comment]
Orca proto change is imported cl/538546518, enabling auto-merge.

**Pull request merged**